pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -315,6 +318,20 @@ func (config *V2RegistriesConf) postProcess() error {
 		config.UnqualifiedSearchRegistries[i] = registry
 	}
 
+	// Registries are ordered and the first longest prefix always wins,
+	// rendering later items with the same prefix non-existent. We cannot error
+	// out anymore as this might break existing users, so let's just ignore them
+	// to guarantee that the same prefix exists only once.
+	prefixMap := make(map[string]bool)
```

Comment:
(Absolutely non-blocking: `prefixExists`? `seenPrefix`? `knownPrefix`?  “map” is fairly redundant, it’s easy enough to see that the type is a `map` within the scope of these <10 lines.)